src/TTGamesExplorerRebirthUI/Forms/TextForm.cs

```diff
@@ -20,6 +20,14 @@ public TextForm(string fileName, byte[] fileBuffer, object datFile = null)
 
             fastColoredTextBox1.BackColor = Colors.DarkBackground;
             fastColoredTextBox1.SelectionColor = Colors.BlueHighlight;
+            
+            if (fileBuffer.Length > 10000000) // 10 MB
+            {
+                fastColoredTextBox1.Text = "No file open";
```

Comment:
I'm not sure this is needed

Reply:
I can't close the form since you call .ShowDialogue(), so I just open a blank file with "no file open" as a placeholder. I'm not sure how to do it better, but you're free to fix that if you know how